core/src/test/java/com/sequenceiq/cloudbreak/core/cluster/ClusterUpscaleServiceTest.java

```diff
@@ -207,7 +207,7 @@ public void testInstallServicesShouldThrowExceptionWhenFailedToRemoveUnusedParce
         when(parcelService.removeUnusedParcelComponents(stackDto)).thenReturn(new ParcelOperationStatus(Map.of(), Map.of("parcel", "parcel")));
 
         CloudbreakException exception = assertThrows(CloudbreakException.class,
-                () -> underTest.installServicesOnNewHosts(1L, Set.of("master"), true, true, new HashMap<>(), null));
+                () -> underTest.installServicesOnNewHosts(1L, Set.of("master"), true, true, new HashMap<>(), null, false));
```

Comment:
Can we have test with verification when the primary gateway change indicated?

Reply:
I have added some tests.